### PR TITLE
Set conditional binary paths for Linux

### LINKED ISSUE #11: Conditionally set paths for Linux
Binary paths differ for Linux, see jryans comment here https://github.com/mozilla/node-firefox-find-simulators/issues/5

### PR DESCRIPTION
**Pull request by brittanystoroz**
Resolves #11

This changes our simulatorBinary properties to methods so we can more easily set conditional paths based on the command being run. Linux paths follow the same pattern for Darwin when requesting a 1.2 or 1.3 simulator, and also split based on `process.arch`.

 I verified tests are passing for Linux x64 but actually can't get my virtual box running for 32-bit. =/ It's just hanging and crashing things and lighting my apartment on fire. Could someone verify all is working as expected there?

### CODE REVIEW

**Comment by tofumatt** [comment]
If the tests pass on 32-bit Linux, is that good enough? :-)

screenshot 2015-02-05 01 16 11

File: lib/platform.js [hunk 1]
```diff
@@ -11,17 +11,44 @@ var firefoxProfilesDir = {
 };
 
 var simulatorBinary = {
-  /*jshint camelcase: false */
-  darwin: {
-    standardPath: 'b2g/B2G.app/Contents/MacOS/b2g-bin',
-    version1_2: 'resources/fxos_1_2_simulator/data/mac64/B2G.app/Contents/MacOS/b2g-bin',
-    version1_3: 'resources/fxos_1_3_simulator/data/mac64/B2G.app/Contents/MacOS/b2g-bin'
+  darwin: function(version) {
```
Comment (tofumatt) [inline lib/platform.js]: Much nicer :+1:

**Comment by tofumatt** [comment]
r+ from me, if this is only WIP because you couldn't get 32-bit Linux working, I ran it and it worked no issues.

**Comment by brittanystoroz** [comment]
Yes that was the only reason for the status of WIP, thanks!

**Pull request merged**